docs/content_management/pages/react_app_block.md

```diff
@@ -103,7 +106,7 @@ Files in this directory create a map of Components which then are imported to `r
 As a result, the components are rendered on the page. 
 
 ``` js
-import Calculator from "../../components/Calculator";
+import Calculator from "assets/page-builder/components/Calculator";
```

Comment:
```
Module build failed: Module not found:                                                                                                                
!!    "./assets/page-builder/react/blocks/Calculator.js" contains a reference to the file "assets/page-builder/components/Calculator".                      
!!    This file can not be found, please check it for typos or update it if the file got moved.
```

To fix the error above, I have to make the following change.

```suggestion
import Calculator from "/assets/page-builder/components/Calculator";
```

Before PR, we had a relative path. This change move to an absolute path.

Reply:
Fixed in #2199